test/performance/odp_ipsecfwd.c

```diff
@@ -561,13 +591,18 @@ static parse_result_t check_options(prog_config_t *config)
 	}
 
 	if (config->num_ifs == 0U) {
-		ODPH_ERR("Invalid number of interfaces: %u, (min: 1, max: %u)\n", config->num_ifs,
+		ODPH_ERR("Invalid number of interfaces: %u (min: 1, max: %u)\n", config->num_ifs,
 			 MAX_IFS);
 		return PRS_NOK;
 	}
 
+	if (config->sa_conf_file != NULL && config->num_sas == 0U) {
+		ODPH_ERR("Invalid SA configuration\n");
```

Comment:
> Additionally, modify SA configuration error checks so that if SA configuration file is passed, then at least one valid SA configuration is expected.

This was slightly annoying, at least to me, as I tried to comment out the last SA of the config file and the test SW protested. I would just remove the check since it is not for safety.

Reply:
Not for safety, but might be less confusing for new users that config file full of invalid data leads to program termination instead of startup. SA config file option can be omitted in case SA config is not needed.